<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploaderGUI</class>
 <widget class="QDialog" name="UploaderGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="fileTypeGB">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Тип файла</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QRadioButton" name="rbCSV">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>.csv</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbXLSX">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>.xlsx</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTXT">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>.txt</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="numberGB">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Количество целевых значений</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QSpinBox" name="spinBox">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextEdit" name="pathTE">
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>20</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pathBtn">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="aceptBtn">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Принять</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancleBtn">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
